bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -7826,7 +7826,7 @@ public short getBattery_Percentage_Start() {
    }
 
    /**
-    * @return Returns battery time in seconds, relative to the tour start time
+    * @return Returns battery time in milliseconds, relative to the tour start time
```

Comment:
>Your comment is correct but the comments of the related fields have still "seconds". I think seconds was intended to be stored but I checked it and milliseconds are stored.

I changed the comment because when I implemented the battery values export, it was confusing because I saw milliseconds for every file so i assumed that it was a bug in the comment.

>This cannot be easily fixed because of the saved tours since it's available. The max value is now roughly 596 hours or 24 days for this battery time field, which is not much but should be OK for 99.9 % use cases.

Maybe you can do a database upgrade to convert from milliseconds to seconds ?

Reply:
> > Your comment is correct but the comments of the related fields have still "seconds". I think seconds was intended to be stored but I checked it and milliseconds are stored.
> 
> I changed the comment because when I implemented the battery values export, it was confusing because I saw milliseconds for every file so i assumed that it was a bug in the comment.
> 
> > This cannot be easily fixed because of the saved tours since it's available. The max value is now roughly 596 hours or 24 days for this battery time field, which is not much but should be OK for 99.9 % use cases.
> 
> Maybe you can do a database upgrade to convert from milliseconds to seconds ?

Yes a database upgrade could fix it but it is no "easily" fixed, I would keep it now as it is.